magma_access_gateway_installer/agw_installer.py

```diff
@@ -52,19 +52,19 @@ def install(self):
     @property
     def _magma_agw_installed(self) -> bool:
         """Checks whether Magma Access Gateway is already installed or not."""
-        return "magma" in check_output(["apt", "list", "--installed"]).decode()
+        return "magma" in check_output(["apt", "-qq", "list", "--installed"]).decode()
 
     @staticmethod
     def update_apt_cache():
         """Updates apt cache."""
         logger.info("Updating apt cache...")
-        check_call(["apt", "update"])
+        check_call(["apt", "-qq", "update"])
 
     @staticmethod
     def update_ca_certificates_package():
         """Updates ca-certificates package"""
         logger.info("Updating ca-certificates package...")
-        check_call(["apt", "install", "ca-certificates"])
+        check_call(["apt", "-qq", "install", "ca-certificates"])
 
     def forbid_usage_of_expired_dst_root_ca_x3_certificate(self):
         """Forbids usage of mozilla/DST_Root_CA_X3.crt certificate."""
```

Comment:
Not related to this line but there is a typo in`repositorys` -> `repositories`

Reply:
Not really a typo - I meant `repository's` :)
I removed the `s` to avoid confusion.